READMEBASE.md

```diff
@@ -31,13 +33,12 @@ To start the development app, follow these commands:
 
 this should start the dev app and rebuild the app for changes
 
-With this, you can start writing content that will be validated on save
+You can also start the development app with readme rewrites enabled by using the command:
+
+        pnpm dev:overwrite
 
-### Todo
+With this, you can start writing content that will be validated on save.
 
-1.  **Testing**
-    Test files within the content folder if it is markdown
-2.  **CI / CD**
-    Create github workflow that will trigger for PR Requests to ensure content follows the linting guide.
+Note: **DO NOT COMMIT THE README FILE**. The readme file will be overwritten once your PR is accepted to the main branch.
```

Comment:
can we add the README file in the .gitignore?

Reply:
if we add README to .gitignore, the github actions bot will also ignore the README file which we do not want since it will be the one to commit the updated README. It allows us to consolidate and avoid merge conflicts when adding new markdown content.